flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/table/Kafka010DynamicSource.java

```diff
@@ -54,7 +59,8 @@
 	 */
 	public Kafka010DynamicSource(
 			DataType outputDataType,
-			String topic,
+			@Nullable List<String> topics,
+			@Nullable Pattern topicPattern,
```

Comment:
Currently, it is very verbose to pass through these to parameters together here and there. An improvement is that we can use `KafkaTopicsDescriptor`, but this can be another issue in the future.